src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -112,7 +112,11 @@ export const OptionsListPopoverActionBar = ({
                 display={showOnlySelected ? 'base' : 'empty'}
                 onClick={() => setShowOnlySelected(!showOnlySelected)}
                 data-test-subj="optionsList-control-show-only-selected"
-                aria-label={OptionsListStrings.popover.getClearAllSelectionsButtonTitle()}
+                aria-label={
+                  showOnlySelected
+                    ? OptionsListStrings.popover.getAllOptionsButtonTitle()
+                    : OptionsListStrings.popover.getSelectedOptionsButtonTitle()
+                }
```

Comment:
I must have somehow grabbed the wrong `aria-label` when refactoring the popover to be multiple smaller components - woops!